airgun/views/common.py

```diff
@@ -385,12 +384,10 @@ class TemplateEditor(View):
         editor = View.nested(TemplateEditor)
 
     """
-    ROOT = ".//div[@class='editor-container']"
-    rendering_options = ToggleRadioGroup(".//div[@class='btn-group']")
-    import_template = Text(".//a[normalize-space(.)='Import']")
-    fullscreen = Text(".//a[normalize-space(.)='Fullscreen']")
-    syntax_type = Select(id='mode')
-    key_binding = Select(id='keybinding')
```

Comment:
These two are still available, but in popover that appears after clicking cog button.
From what I see, nothing in Robottelo refers to these items. We can add support for them in Arigun sometime later.

Reply:
This will be a bit more difficult to add. Considering that these two (+ one new - theme) won't be used in automation I think we don't need these options. Spending time on something we don't even use is not so effective.